qemu/tests/cfg/block_resize.cfg

```diff
@@ -10,9 +10,7 @@
     create_image_stg = yes
     force_create_image_stg = yes
     blk_extra_params_stg = "serial=TARGET_DISK0"
```

Comment:
Yes, this problem has been found on ppc.
But I think getting corresponding disk via serial number is a more reliable way.
And I prefer add the param to all related cases.